docs/src/pages/[platform]/components/collection/react.mdx

```diff
@@ -20,207 +22,176 @@ A Collection wraps Flex and Grid components, and provides a way to display items
-        description:
-          'Three hours north of Auckland, this area features over 144 islands to explore.',
-      },
-      {
-        title: 'Queenstown, South Island',
-        description:
-          "This hopping town is New Zealand's adventure capital and is located right on Lake Wakatipu.",
-      },
-    ]}
-    gap="1.5rem"
-  >
-    {(item, index) => (
-      <Card key={index} padding="1rem">
-        <Heading level={4}>{item.title}</Heading>
-        <Text>{item.description}</Text>
-      </Card>
-    )}
-  </Collection>
+  <DefaultCollectionExample />
+  <ExampleCode>
+
+```tsx file=./examples/DefaultCollectionExample.tsx
+
+```
+
+  </ExampleCode>
 </Example>
 
 ## Collection types
 
-Collection `type` options include `list`, `grid`, and `table`.
+Collection `type` options include `list` and `grid`.
 
 ### List
 
 The `list` collection type can be customized with any of following Flex props: `alignItems`, `alignContent`, `direction`, `gap`, `justifyContent`, `wrap`.
 
-```jsx
-<Collection
-  type="list"
-  items={visitNewZealand}
-  gap="1.5rem"
-  direction="row"
-  justifyContent="space-between"
-  wrap="wrap"
->
-  {(item, index) => (
-    <Card key={index} padding="1rem">
-      <Heading level={4}>{item.title}</Heading>
-      <Text>{item.description}</Text>
-    </Card>
-  )}
-</Collection>
+<Example>
+  <ListCollectionExample />
+
+  <ExampleCode>
+
+```tsx file=./examples/ListCollectionExample.tsx
+
 ```
 
+  </ExampleCode>
+</Example>
+
+### Grid
+
+The `grid` collection type can be customized with and of the following Grid props: `templateColumns` and `templateRows`. Then the Collection children can use the Grid props to control their size and placement such as `row` and `column`.
+
 <Example>
-  <Collection
-    type="list"
-    items={[
-      {
-        title: 'Fiordland National Park',
-        description:
-          'This national park includes the famous fjords of Milford, Dusky and Doubtful Sounds.',
-      },
-      {
-        title: 'Bay of Islands, North Island',
-        description:
-          'Three hours north of Auckland, this area features over 144 islands to explore.',
-      },
-      {
-        title: 'Queenstown, South Island',
-        description:
-          "This hopping town is New Zealand's adventure capital and is located right on Lake Wakatipu.",
-      },
-    ]}
-    direction="row"
-    justifyContent="space-between"
-    wrap="wrap"
-  >
-    {(item, index) => (
-      <Card key={index} padding="1rem" maxWidth="180px" fontSize="0.5em">
-        <Heading level={4}>{item.title}</Heading>
-        <Text>{item.description}</Text>
-      </Card>
-    )}
-  </Collection>
+  <GridCollectionExample />
+  <ExampleCode>
+
+```tsx file=./examples/GridCollectionExample.tsx
+
+```
+
+  </ExampleCode>
 </Example>
 
 ## Pagination
 
 A Collection can be paginated, adding a special `isPaginated` property. Change the page size passing a `itemsPerPage` property (default: 10).
```

Comment:
nit: "A Collection can be paginated by adding a special `isPaginated` property. Change the page size by passing a `itemsPerPage` property (default: 10).